Src/AutoFixture.NUnit2.UnitTest/FrozenAttributeTest.cs

```diff
@@ -31,12 +33,15 @@ public void GetCustomizationFromNullParamterThrows()
         }
 
         [Test]
-        public void GetCustomizationReturnsCorrectResult()

```

Comment:
I moved the discussion to a new [pull request](https://github.com/AutoFixture/AutoFixture/commit/ae16349568323846c883e7a34c481714d692ce7a#commitcomment-13475877).